client/src/components/common/boost/BoostCalibration.tsx

```diff
@@ -0,0 +1,86 @@
+import React , {useCallback, useState} from 'react';
+import {Button, Card, InputGroup, FormControl} from 'react-bootstrap';
+import {setCalibration as sendCalibrationValue, resetCalibration} from 'api/common/powerModel';
+// import {useSensorData} from 'api/v2/sensors/data';
+
+/**
+ * Boost distance claibration input form
+ * 
+ * @returns {React.Component} Component
+ */
+function BoostCalibrationInput() {
+    const [calibrationValue, setCalib] = useState('');
+
+    const handleCalibrationChange = useCallback(
```

Comment:
We seem to use `useCallback` a lot around here but I don't believe it's necessary. (anyone correct me if you believe otherwise...) These functions can (I think) just be normal callbacks.

Reply:
I think the difference is quite minimal but since it is being used in the form that is re-rendered (I think) when used, having `useCallback` makes it a bit more efficient.

Reply:
I tried removing the `useCallback` but it generates a warning from `yarn`, so at the moment I'm keeping them as is, but let me know if it's possible to remove those warnings and whether we want to avoid using `useCallbacks` where it's not necessary

Reply:
On second thoughts I think Nick is right. Don't worry about this :)